plugins/modules/pnp_intent.py

```diff
@@ -611,36 +629,41 @@ def get_have(self):
                             self.msg = "Installation mode must be in \
                             INSTALL mode to upgrade the image. Current mode is\
                             {0}".format(install_mode)
+                            self.log(str(self.msg), "CRITICAL")
                             self.status = "failed"
                             return self
 
                         have["image_id"] = image_list[0].get("imageUuid")
-                        self.log("Image Id: " + str(have["image_id"]))
+                        self.log("Image Id is {0}".format(str(have["image_id"])), "INFO")
 
                     template_name = self.want.get("template_name")
                     if template_name:
                         if not (template_list and isinstance(template_list, list)):
                             self.msg = "Project Not Found \
                                 or Project is Empty"
+                            self.log(self.msg, "CRITICAL")
                             self.status = "failed"
                             return self
 
                         template_details = get_dict_result(template_list, 'name', template_name)
                         if template_details:
                             have["template_id"] = template_details.get("templateId")
                         else:
-                            self.msg = "Template Not found"
+                            self.msg = "Template {0} Not found".format(template_name)
```

Comment:
self.msg = "Template '{0}' is not found.".format(template_name)

Reply:
Resolved